frontends/web/src/i18n/i18n.ts

```diff
@@ -91,9 +91,6 @@ i18Init.init({
   }
 });
 
-// load the default language first
-loadLanguage(defaultLang);
```

Comment:
This is the relevant change: in this commit I changed @cstenglein initial default loading with a initialized callback see below. cc @shonsirsha